src/index.ts

```diff
@@ -44,3 +46,10 @@ mongoose
 if (process.env.ENABLE_DISCORD_BOT === 'true' && process.env.DISCORD_TOKEN !== undefined) {
     initDiscord();
 }
+
+// Scheduling an autodelete task that runs every day at 00:00
+schedule.scheduleJob('*0 0 * * *', async function () {
+    const currDate = new Date();
+    currDate.setDate(currDate.getDate() - 7);
```

Comment:
-7 will not work if it's the same month, I'd use the already imported Luxon's ability to compare between dates. Also I would just call it date and not currDate because it's a week ago